tester_app/client/assets/stylesheets/_app-styling-post-bootstrap-loading.scss

```diff
@@ -0,0 +1,4 @@
+// This file is used as part of the bootstrap-sass-loader customization
+// Included from bootstrap-sass.config.js
+// Use it to apply styles that need to go AFTER bootstrap has already loaded
+// For example, you can add your own styles and override bootstrap styles.
```

Comment:
// This is where you can add your own styles and override bootstrap styles, utilizing bootstrap variables.